app/main.py

```diff
@@ -2,0 +1,34 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hiden: bool = False) -> None:
+        self.health = health
+        self.hidden = hiden
```

Comment:
The attribute `self.hidden` is being assigned the value of `hiden`, which is a typo. It should be `hidden` to match the intended functionality.